### ISSUE TITLE
Pick plugin filename and identifier from manifest instead of package.json

### ISSUE THREAD

**Issue by pratyushtewari** [skpm/skpm#230]
How can I create different plugins from the same package using multiple manifest files?

I am using different manifest to create two plugins using 

`npm run skpm-build --manifest=src/manifest-one.json`
and 
`npm run skpm-build --manifest=src/manifest-two.json`

The problem is both are using the name from my package.json instead of the manifest file. 
Here in this code: 
https://github.com/skpm/skpm/blob/75685f2a32e5677f09a32482466a23bb061c9e43/packages/utils/skpm-config.js#L14

we probably need to change this to
```
identifer: argv.manifest.identifier || skpmConfig.identifier || packageJSON.name,
```
I can only use the package.name or package.skpm.identifier which will be same for all the plugins I create from this package.json.

Is there any way to use the identifier from my manifest file?

**Comment by mathieudutour** [comment]
The line you referenced is only part of the story.
The builder then merge the "skpm config" coming from the package.json and the manifest.json, giving precedence to the manifest.json.

Also, `argv.manifest` is a string, not the content of the file

**Comment by pratyushtewari** [comment]
I get it,
How do I make this read the identifier from my manifest instead of package.name?

**Comment by mathieudutour** [comment]
What do you mean by "make this read"? 
The builder should already use the identifier from the manifest if there is one: https://github.com/skpm/skpm/blob/master/packages/builder/src/build.js#L107

**Comment by pratyushtewari** [comment]
Ah, I was not clear enough. I want to create two different plugins.
I tried adding the skpm.main in my manifest but the skpm-build is only looking in the package.json for the `skpm.main`.

So when I remove
```
  "skpm": {
    "main": "my-plugin-name.sketchplugin"
  },
```
from my package.json, I get
```
error Missing "skpm.main" fields in the package.json. Should point to the ".sketchplugin" file
```

Is there any other way to pass or overwrite the name of the built sketch plugin bundle using the manifest file?

**Comment by mathieudutour** [comment]
ok so the `main` field and not the `identifier` :) That makes more sense. I'll have a fix soon 👍

**Comment by pratyushtewari** [comment]
I'll re-word the title of the issue to align it more.

**Comment by mathieudutour** [comment]
In 0.7.2, there is a new `output` option that you can use to specify the path to the built plugin (eg. the `main` field)

**Comment by pratyushtewari** [comment]
This worked wonderfully!!! Thanks.

So I run 
```
skpm-build --manifest=<location of manifest.jon> --output=<pluginname>.sketchplugin
```
it would be nice to not get 
```
error Missing "skpm.main" fields in the package.json. Should point to the ".sketchplugin" file
```
since now I can use the --output parameter.

**Comment by pratyushtewari** [comment]
@mathieudutour , now that I can create two plugins, how do I use the `skpm-link` to create the link to sketch now.
https://github.com/skpm/skpm/blob/master/packages/builder/src/link.js only reads the `skpm.main` from package.json.

**Comment by bomberstudios** [comment]
Hi @pratyushtewari, could you please open new issues for these two last requests? It's easier for us to keep track of these things that way. Thanks!

**Comment by mathieudutour** [comment]
Ah @pratyushtewari fixed it already: #232
